modules/basic-auth/config/BASIC_AUTH_IP_WHITELIST.ts

```diff
@@ -0,0 +1 @@
+export default process.env.BASIC_AUTH_IP_WHITELIST || '93.185.99.21|172.18.0.1'
```

Comment:
What are those IP addresses? Remove them and just `export default []` (no need to load values from env)